src/app/routes/enrolment/my-enrolment-list/my-enrolment-list.component.ts

```diff
@@ -93,6 +93,15 @@ export class MyEnrolmentListComponent implements OnInit, OnDestroy {
     return !element?.isSynced;
   }
 
+  showCancelClaimOption(element: EnrolmentClaim) {
+    return (
+      !element?.isAccepted &&
+      !element?.isRejected &&
+      !element.isRevokedOffChain &&
+      !element.isRevokedOnChain
```

Comment:
I would add the elvis (`?`) to avoid possible "cannot read property x of undefined" errors if these are reordered or the first couple are removed.
```suggestion
      !element?.isRevokedOffChain &&
      !element?.isRevokedOnChain
```

Reply:
Why not to put this into EnrolmentClaim object? The reason i created this object was that we had a lot of copy pasted logic like this in components.
I mean, you could add method `canCancelClaim(): boolean`
Anyway isAccepted and isRejected is always available. No need to put elvis there.
Another thing it is good to add some tests.
@jrhender there will be no `"cannot read property x of undefined"` because the object (element) is available.

Reply:
@dawidgil  It wasn't in the Enrolment claim object before today. I assumed that you had moved what you wanted into the object when you created it.

Reply:
@whitneypurdum i might miss something always :D but do you agree that this might fits there or not really?

Reply:
@dawidgil Implemented the above in claim class